src/lava/proc/lif/models.py

```diff
@@ -97,17 +93,29 @@ def __init__(self):
         self.vth_shift = 6
         self.act_shift = 6
 
-    def run_spk(self):
-        # Receive synaptic input
-        a_in_data = self.a_in.recv()
+    def scale_bias(self):
+        """Scale bias with bias exponent by taking into account sign of the
+        exponent"""
+        self.effective_bias = np.where(self.bias_exp >= 0, np.left_shift(
+            self.bias, self.bias_exp), np.right_shift(self.bias,
+                                                      -self.bias_exp))
+        # self.effective_bias = np.left_shift(self.bias, self.bias_exp) if \
+        #     self.bias_exp >= 0 else np.right_shift(self.bias, -self.bias_exp)
+        self.isbiasscaled = True
```

Comment:
Can we comment this out for now. This does not allow interacting with bias during runtime as we discussed earlier.

Reply:
Fixed. For now, we will scale bias at every time-step in the case of fixed point ProcessModel. This will be slow, but it will work. When we make a change in Var.set() API, we can change this behaviour back again.